core/src/test/java/org/elasticsearch/indices/IndicesRequestCacheTests.java

```diff
@@ -91,51 +92,14 @@ public void testBasicOperationsCache() throws Exception {
         assertEquals(1, requestCacheStats.stats().getHitCount());
         assertEquals(1, requestCacheStats.stats().getMissCount());
         assertEquals(0, requestCacheStats.stats().getEvictions());
-        assertEquals(1, entity.loaded);
+        assertTrue(entity.loadedFromCache());
         assertEquals(0, cache.count());
         assertEquals(0, requestCacheStats.stats().getMemorySize().bytesAsInt());
 
         IOUtils.close(reader, writer, dir, cache);
         assertEquals(0, cache.numRegisteredCloseListeners());
     }
 
-    public void testCacheWithDifferentEntityInstance() throws Exception {
```

Comment:
It started testing the same thing as the test above it once I started using AbstractIndexShardCacheEntity.